src/backend/didConfiguration/domainLinkageCType.ts

```diff
@@ -20,9 +21,16 @@ export async function storeDomainLinkageCType(): Promise<void> {
   });
 
   if (await CTypeUtils.verifyStored(draft)) {
-    logger.info('Domain Linkage CType is already on the blockchain');
+    if (configuration.storeDidAndCTypes) {
+      logger.info('Domain Linkage CType is already on the blockchain');
+    }
     return;
   }
+
+  if (!configuration.storeDidAndCTypes) {
```

Comment:
this can't be the `else` branch of the same condition above?

Reply:
Why use `else` if the previous branch did a `return`? I think even an eslint rule advises against that.